src/mdns/mdns.cpp

```diff
@@ -197,6 +197,20 @@ void Publisher::OnServiceResolved(std::string aType, DiscoveredInstanceInfo aIns
                 aInstanceInfo.mRemoved ? "remove" : "add", aInstanceInfo.mName.c_str(), aInstanceInfo.mHostName.c_str(),
                 aInstanceInfo.mAddresses.size());
 
+    if (!aInstanceInfo.mRemoved)
+    {
+        std::string addressesString;
+        for (const auto &address : aInstanceInfo.mAddresses)
```

Comment:
nit: empty line after declaration
```suggestion
        std::string addressesString;

        for (const auto &address : aInstanceInfo.mAddresses)
```